pages/management/ManagementHandler.inc.php

```diff
@@ -217,6 +223,7 @@ function workflow($args, $request) {
 
 		$authorGuidelinesForm = new \PKP\components\forms\context\PKPAuthorGuidelinesForm($contextApiUrl, $locales, $context);
 		$metadataSettingsForm = new \APP\components\forms\context\MetadataSettingsForm($contextApiUrl, $context);
+		$allowSubmissionsForm = new \APP\components\forms\context\AllowSubmissionsForm($contextApiUrl, $context);
```

Comment:
We'll want to use this form in OMP and OPS, so I think you can move it to `pkp-lib` and change the namespace to `\PKP\...`.